tests/unit/test_magma_access_gateway_installer.py

```diff
@@ -50,3 +51,21 @@ def test_given_invalid_ip_address_cli_argument_is_passed_when_validate_args_then
 
         with self.assertRaises(ValueError):
             magma_access_gateway_installer.validate_args(test_args)
+
```

Comment:
we should probably have tests that validate that the dns was properly configured with the provided DNS values

Reply:
In **test_agw_network_configurator.py** we already have following DNS tests:
`test_given_dns_is_already_configured_when_configure_dns_then_no_config_is_written`
`test_given_dns_is_not_correctly_configured_when_configure_dns_then_dns_config_is_written`
`test_given_dns_is_not_correctly_configured_when_configure_dns_then_dns_service_is_restarted`

Last one is irrelevant, but first two actually cover DNS configuration with given addresses. Unless I'm missing something, the only added value of new test here would be checking whether DNSs list is correctly passed to the constructor of `AGWInstallerNetworkConfigurator` class. And it would require a lot of mocking :)